device:
  mirte:
    type: pcb
    version: 0.6
    max_frequency: 50
modules:
  motorservocontroller:
    device: mirte
    type: PCA9685
    addr: 0x40
    connector: I2C1
    frequency: 1500
    motors:
      left_front:
        pin_A: 8
        pin_B: 9
      left_rear:
        pin_A: 10
        pin_B: 11
      right_front:
        pin_A: 12
        pin_B: 13
      right_rear:
        pin_A: 14
        pin_B: 15
    servos:
      servo1:
        pin: 1
  servo_controller:
    device: mirte
    type: Hiwonder_Servo
    uart: 1  # FIXME: Remove
    pins:
      rx: 4
      tx: 5
    servos:
      servoH1:
        id: 3
        min_angle: 9000
        max_angle: 14000
      servoH2:
        id: 4
        min_angle: 800
        max_angle: 4000
  power_watcher:
    device: mirte
    type: INA226
    connector: I2C1
    addr: 0x41
    min_voltage: 11
    max_current: 1
    max_voltage: 15